store/tikv/error.go

```diff
@@ -65,3 +66,7 @@ func init() {
 	}
 	terror.ErrClassToMySQLCodes[terror.ClassTiKV] = tikvMySQLErrCodes
 }
+
+const (
+	codeErrorTokenLimit terror.ErrCode = 1
```

Comment:
Should we define an error code in parser?

Reply:
Yes, please add the error code/name in parser.